Preliminary commit to make comparisons easier for ACIS Ops

Gregg140 opened the pull request:
Provides the file comparison.

Commit:
Preliminary commit to make comparisons easier for ACIS Ops

cegrant commented:
line 14-15 - the DPA-A anomaly page formats these differently, with separate lines and the second line in italics.  I think this page should be the same.
line 39 - MJF isn't an acronym we use regularly, simpler to write it out.
line 93 - typo in "otherewise"
lines 99-103 - I can't see a scenario where you would need an additional WSPOW00000 command in your CAP, I think a WSPOW0002A would cover all circumstances. 
lines 99-103 - typo "shuitdown"
lines 99-103 - both commands need to be formatted to look like the commands in the DPA-A anomaly page
lines 99-103 - WSPOW0002A needs the CLD file name listed, then later on a "CLD Scripts" section with the link to occweb -- again you can model it on the DPA-A anomaly page.
lines 99-103 - this is a bigger meta question but I worry about how much time the recovery might be taking.  Leaving off the WSPOW0002A step is not an immediate health and safety risk.  I feel like that should somehow come through here, but I'd be interested in what other acisdudes think.
line 183 - typo "Relevan"

jazan12 commented on source/dpab_shutdown.rst:
```diff
@@ -170,6 +179,15 @@ CAPs
 .. _mptl: http://cxc.cfa.harvard.edu/acis/acispy/command_line.html#multiplot-tracelog
 
+
+Relevan Notes/Memos
```
Change to *Relevant*

jazan12 commented on source/dpab_shutdown.rst:
```diff
@@ -35,9 +36,10 @@ Within a major frame (32.2 seconds), one should see:
 * DPA-B POWER should go to zero
 * 1DP28BVO (DPA-B +28V Input Voltage) is expected to have a small uptick, ~0.5 V, consistent with
-  the load suddenly dropping to zero
+  the load suddenly dropping to zero. May require more than 1 MJF.
```
Suggest "major frame"

Commit:
Typos fixed; Notes section updated

jzuhone commented:
I may fiddle with the formatting a bit after we merge this in, but otherwise I think this is fine.

cegrant commented:
This still needs the CLD file name and the link.  I think that's pretty important if we're asking folks to put it in the CAP.  And I'd much prefer if it made clear that WSPOW0002A is the power command you would be running since I don't think there's any scenario where WSPOW00000 would be appropriate.

Gregg140 commented:
If you would read the note written when I pushed the latest change up 
you would have seen this:

ggermain
|Typos fixed; Notes section updated 
<https://github.com/acisops/acis_anomaly_pages/pull/24/commits/07547bfaee02c7908d09e2ab8986d859671e8d8e>
|

|This means those are the only changes I made.|
|
|
|That doesn't mean those are the only changes I'm going to make|
|
|
||
||
On 10/9/19 1:17 PM, cegrant wrote:
>
> This still needs the CLD file name and the link. I think that's pretty 
> important if we're asking folks to put it in the CAP. And I'd much 
> prefer if it made clear that WSPOW0002A is the power command you would 
> be running since I don't think there's any scenario where WSPOW00000 
> would be appropriate.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/acisops/acis_anomaly_pages/pull/24?email_source=notifications&email_token=AANFYCWVTBD52RXUKKEUHV3QNYG4HA5CNFSM4I7A2LK2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEAYUKMA#issuecomment-540099888>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AANFYCX7ETNWGV4Y7BFRTN3QNYG4HANCNFSM4I7A2LKQ>.
>

--

cegrant commented:
I did see that and then John approved it, so I thought we were going ahead as is.  Sorry if I jumped the gun.

jzuhone commented:
@cegrant sorry, I may have jumped the gun myself a bit.

Commit:
Further updates as suggested by ACIS Ops team

Commit:
Further updates as suggested by ACIS Ops team

cegrant reviewed (commented):
I'm happy with the updates. The link for the CLD to occweb is not crucial, so shouldn't hold this up.  I'm just commenting, but if you're ready for approvals I'm happy to do that too.

Commit:
Added OBC B-side entried for the DPAA and DPAB shutdown anomalies in the main page

Gregg140 commented:
Ok I've modified the anomaly main page (index.rst) to show placeholders for future links (OBC B-side)
And cleaned up the CLD file entry for the dpab_shutdown.rst

Ready for final approval.

Thanks

Commit:
Fixed OBS mislabel on the anomaly main page.

Gregg140 commented:
Dopey moi. Fixed

jazan12 reviewed (approved):
Looks good!
